spark/sub_overlap_pipeline.py

```diff
@@ -94,8 +94,8 @@ def filter_top_subreddits(df_reddit, subs_to_incl):
     )
 
     # Take the top X rows and convert them to list
-    df_top_subs = df_subred_count.limit(subs_to_incl)
-    top_subs = df_top_subs.select("subreddit").rdd.flatMap(lambda x: x).collect()
+    # df_top_subs = df_subred_count.limit(subs_to_incl)
+    top_subs = df_subred_count.select("subreddit").rdd.flatMap(lambda x: x).collect()
 
     # Filter the original dataframe using the list
     df_sub_filtered = df_reddit.filter(col("subreddit").isin(top_subs))
```

Comment:
Here it looks like the filtering is still taking place (compare to the PR description).